<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvConfigUI</class>
 <widget class="QWidget" name="TvConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TvConfigUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tvCard">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>TV Kartı</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="cardManufacturer">
           <property name="text">
            <string>Üretici</string>
           </property>
          </widget>
          <widget class="QLabel" name="cardModel">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QListWidget" name="cardManList">
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
          <widget class="QListWidget" name="cardModList">
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tuner</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="cardManufacturer_2">
           <property name="text">
            <string>Üretici</string>
           </property>
          </widget>
          <widget class="QLabel" name="cardModel_2">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QListWidget" name="cardManList_2">
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
          <widget class="QListWidget" name="cardModList_2">
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Seçenekler</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Phase Locked Loop(PLL)</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>Do not use PLL</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>28 Mhz Crystal</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>35 Mhz Crystal</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Eklentiler</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Radyo Kartı</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
